pages/docs/reference/collection-aggregate.md

```diff
@@ -10,7 +10,7 @@ title: "Aggregate Operations"
 Kotlin collections contain functions for commonly used _aggregate operations_ – operations that return a single value based on the collection content.
 Most of them are well known and work the same way as they do in other languages:
 
-* [`min()`](/api/latest/jvm/stdlib/kotlin.collections/min.html) and [`max()`](/api/latest/jvm/stdlib/kotlin.collections/max.html) return the smallest and the largest element respectively;
+* [`minOrNull()`](/api/latest/jvm/stdlib/kotlin.collections/min-or-null.html) and [`maxOrNull()`](/api/latest/jvm/stdlib/kotlin.collections/max-or-null.html) return the smallest and the largest element respectively;
```

Comment:
Here and below:
I guess you should change the description here (or add a note?) about returning null on empty collections?